x-pack/plugin/esql/qa/server/src/main/java/org/elasticsearch/xpack/esql/qa/rest/AllSupportedFieldsTestCase.java

```diff
@@ -976,7 +1005,7 @@ private static boolean supportedInEnrich(DataType t) {
             // https://github.com/elastic/elasticsearch/issues/127350
             case AGGREGATE_METRIC_DOUBLE, SCALED_FLOAT,
                 // https://github.com/elastic/elasticsearch/issues/139255
```

Comment:
I wonder if we need to update https://github.com/elastic/elasticsearch/issues/139255. Left a comment there.